samza-core/src/main/java/org/apache/samza/config/DefaultChooserConfig.java

```diff
@@ -30,15 +30,17 @@
  * A convenience class for fetching configs related to the {@link org.apache.samza.system.chooser.DefaultChooser}
  */
 public class DefaultChooserConfig extends MapConfig {
-  public static final String BOOTSTRAP = StreamConfig.STREAM_PREFIX() + "samza.bootstrap";
-  public static final String PRIORITY = StreamConfig.STREAM_PREFIX() + "samza.priority";
-  public static final String BATCH_SIZE = "task.consumer.batch.size";
+  private static final String BOOTSTRAP = StreamConfig.SAMZA_PROPERTY() + "bootstrap";
```

Comment:
I prefer at least moving the reading config part to StreamConfig. This class does provides a few util methods there (the class name is pretty misleading), so either leaves them here or inline them directly in the default chooser(). Probably deleting the class is cleaner.